src/shared/protocol/game_events.c

```
@@ -483,3 +484,44 @@ bool bc_parse_delete_player_anim(const u8 *payload, int len,
+ * the exploding-event factory, so a caller can pass every inbound 0x06 payload
+ * here as a cheap discriminator.
+ */
+bool bc_parse_python_exploding_event(const u8 *payload, int len,
+                                     bc_exploding_event_t *out)
+{
+    memset(out, 0, sizeof(*out));
+
+    bc_buffer_t buf;
+    bc_buf_init(&buf, (u8 *)payload, (size_t)len);
+
+    u8 opcode;
```

Clang (14.0.6)

[warning] variable 'opcode' is not initialized

(cppcoreguidelines-init-variables)